.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: android
+
+jdk: oraclejdk8
+
+android:
+  components:
+    - tools
+    - android-25
+    - build-tools-25.0.1
+    - platform-tools
+    - extra-android-m2repository
+
+after_failure:
+    - cat $TRAVIS_BUILD_DIR/viewpump/build/reports/lint-results.xml || true
```

Comment:
Nice. You can also selectively enable lintOptions to output to stdout like [this](https://github.com/hzsweers/empty-android-project/blob/master/app/build.gradle#L77-L87)